components/action-mgt/org.wso2.carbon.identity.action.execution/src/test/java/org/wso2/carbon/identity/action/execution/ActionExecutorServiceImplTest.java

```diff
@@ -389,6 +433,20 @@ private ActionInvocationResponse createSuccessActionInvocationResponse() throws
         return actionInvocationResponse;
     }
 
+    private ActionInvocationResponse createFailureActionInvocationResponse() {
+
+        ActionInvocationFailureResponse failureResponse = mock(ActionInvocationFailureResponse.class);
+        when(failureResponse.getActionStatus()).thenReturn(ActionInvocationResponse.Status.ERROR);
```

Comment:
This should be 'FAILED'?

Reply:
Addressed with commit 0812bd15f91ba1f366b024b26a5aa4e8178d752f